src/System.Security.Cryptography.Cng/tests/OpenTests.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class OpenTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void OpenFromHandle1()
+        {
+
+            byte[] reExported;
+            using (CngKey keyOriginal = CngKey.Import(TestData.Key1, CngKeyBlobFormat.GenericPublicBlob))
+            {
+                SafeNCryptKeyHandle keyHandle = keyOriginal.Handle;
+                Assert.NotNull(keyHandle);
+                CngKey key = CngKey.Open(keyHandle, CngKeyHandleOpenOptions.EphemeralKey);
+                keyHandle.Dispose();
+
+                Assert.Equal(CngAlgorithm.ECDiffieHellmanP256, key.Algorithm);
```

Comment:
It feels like really these should be comparing keyOriginal to key, not key to "the right answer" (since "the right answer" was already a different test)